chatbot.js

```diff
@@ -17,16 +18,11 @@ const karma = require('./services/karmaService.js');
 const DEPLOY_FLAG = process.env.DEPLOY;
 const TEST_FLAG = process.env.TEST;
 
-// Main function which handles the user input and decides what needs to be done.
-const replyToUser = (user, message) => {
-  const username = user.username;
-
-  if (message.startsWith(constants.BOT_MENTION_NAME)) {
-    const parsedMessage = message.slice(constants.BOT_MENTION_NAME.length + 1);
-    const startsWithString = utils.getStartsWith(parsedMessage);
-    const cityName = parsedMessage.substr(parsedMessage.indexOf(' ') + 1);
 
-    if (startsWithString === constants.BOT_ACTIONS.HELP) {
+const callSpecificService = (username, message, parsedMessage) => {
```

Comment:
I think you missed my earlier comment about renaming this function to something more descriptive and clear.